js_modules/dagit/packages/core/src/instance/DeploymentStatusProvider.tsx

```diff
@@ -35,28 +27,10 @@ export const DeploymentStatusProvider: React.FC<Props> = (props) => {
 
   const codeLocations = useCodeLocationsStatus(!include.has('code-locations'));
   const daemons = useDaemonStatus(!include.has('daemons'));
-  const autoMaterialization = useAutoMaterializationStatus();
```

Comment:
was this removed intentionally?

Reply:
yes- https://elementl-workspace.slack.com/archives/C04UD72HHQX/p1681759584001969?thread_ts=1681506142.934559&cid=C04UD72HHQX